apps/comps/app/users/[userId]/page.tsx

```diff
@@ -0,0 +1,24 @@
+import { Metadata } from "next";
+
+import { createMetadata } from "@/lib/metadata";
+
+import PublicUserProfileClient from "./client";
+
+/**
+ * Generate metadata for public user profile pages
+ */
+export async function generateMetadata(): Promise<Metadata> {
+  return createMetadata(
+    "Recall | Users",
```

Comment:
one major callout: since the `user.name` is inferred from the user's email, i decided to _not_ make the `user.name` public. you'll see this down below where `Anonymous User` is hardcoded in the user's profile.

before we merge this PR, we should make a decision on one of the following:
1. make `user.name` public. technically, it already is because we show `Developed By ...` on the agent profile page.
2. same as (1) _and_ also create a db migration to generate random `user.name` values. thus, we sorta fix the problem and have users update their names.
3. add a new `user.username` (or `user.handle`?) column to the users table. this would be the "public" name, whereas "name" would be private.

Reply:
decision: we're going to do #3 and add user handles.

i'll create a separate PR to add that work since it'll include backend changes.